### ISSUE TITLE
Color variables for success and warning lack contrast

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/nextcloud-vue#2858]
The color values for both `--color-success` and `--color-warning` lack contrast for even large text AA and graphical objects & interface components – meaning we should not use them anywhere.
Darkening them a bit would make them usable at least for those 2 cases. What do you think @juliushaertl @skjnldsv @marcoambrosini @juliakirschenheuter?

color-success|color-success darkened
-|-
![color success](https://user-images.githubusercontent.com/925062/179731789-efa0da49-2b93-42d8-bee5-a49281542901.png)|![color success darker](https://user-images.githubusercontent.com/925062/179731783-b08fa83d-4893-4e1b-b034-9fdcfba41ac2.png)

color-warning|color-warning darkened (probably needs adjustment, dark orange always looks a bit bad ;)
-|-
![color warning](https://user-images.githubusercontent.com/925062/179731796-b6f01cc7-785a-4288-9f1f-230852ac3351.png)|![color warning darker](https://user-images.githubusercontent.com/925062/179731791-fb718efc-942a-4cf8-8ffd-de642d1bc0ce.png)

**color-error works well**
![color error](https://user-images.githubusercontent.com/925062/179731794-24c0fc0f-9788-47f0-98a0-6818f64f7b61.png)
